src/main/java/core/basesyntax/operationstrategy/BalanceOperationHandler.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.operationstrategy;
+
+import core.basesyntax.model.TransactionDto;
+
+public class BalanceOperationHandler implements OperationHandler {
+    @Override
+    public Integer doOperation(TransactionDto fruitRecord) {
+        int fruitsBalance = fruitRecord.getFruitAmount();
+        if (fruitsBalance < 0) {
```

Comment:
but you have validator in the reading file method?

Reply:
Done